modules/app_lb/main.tf

```diff
@@ -18,14 +18,31 @@ locals {
   redirect_configuration_name    = "${var.network.name}-rdrcfg"
 }
 
+resource "azurerm_web_application_firewall_policy" "default" {
+  name                = "${var.namespace}-wafpolicy"
+  resource_group_name = var.resource_group_name
+  location            = var.location
+
+  managed_rules {
+    managed_rule_set {
+      version = "3.2"
+    }
+  }
+
+  tags = var.tags
+}
+
 resource "azurerm_application_gateway" "default" {
   name                = "${var.namespace}-ag"
   resource_group_name = var.resource_group_name
   location            = var.location
 
+  tags               = var.tags
+  firewall_policy_id = azurerm_web_application_firewall_policy.default.id
+
   sku {
-    name = "Standard_v2"
-    tier = "Standard_v2"
+    name = "WAF_v2"
+    tier = "WAF_v2"
```

Comment:
If we use `WAF` we have to [disable a bunch of rules ](https://github.com/wandb/local/blob/main/terraform/azure/infra/main.tf#L118-L155) as it falsely flags a ton of requests.  It's also pretty expensive.  If we can find a way to avoid it, probably ideal.

Reply:
I've reverted to Standard_v2, we can punt on this